### PR TITLE
Refactor/search functionality

### PR DESCRIPTION
**Pull request by lanehunter-dev**
#### What's this PR do?
- Adds functionality for searching by recipe name or ingredient list.
- Reformatted recipe data after fetch so ingredients have names 
#### Where should the reviewer start?
script.js 
#### How should this be manually tested?
Search for ingredients that exist in ingredient lists but not in recipe names, ensure your search term brought up the proper cards
#### Questions:
Will it be possible to completely get rid of querySelectors?

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -24,11 +24,35 @@ const loadPageHandler = () => {
   findTags(recipeData)
 }
 
+const reformatData = () => {
```
Comment (dreardon1021) [inline src/scripts.js]: This is dope. also noticed the awesome thing your doing that makes this happen on load pretty much.

File: src/scripts.js [hunk 2]
```diff
@@ -52,7 +76,6 @@ $('.filter-btn').click(findCheckedBoxes);
 $('main').click(selectCard);
 $(".my-pantry-btn").click(toggleMenu);
 $(".saved-recipes-btn").click(showSavedRecipes);
-$(".search-btn").click(searchRecipes);
```
Comment (dreardon1021) [inline src/scripts.js]: Should we remove the search button completeley and just make it an image or something?

File: src/scripts.js [hunk 3]
```diff
@@ -167,6 +190,7 @@ function showSavedRecipes() {
 function openRecipeInfo(recipeId) {
   domUpdates.display('.recipe-instructions')
   let recipe = recipeData.find(recipe => recipe.id === Number(recipeId));
+  console.log(recipe);
```
Comment (dreardon1021) [inline src/scripts.js]: remove this console log

File: src/scripts.js [hunk 4]
```diff
@@ -24,11 +24,35 @@ const loadPageHandler = () => {
...
+        })
+        return reformattedList
+      }, []),
+      instructions: recipe.instructions,
+      tags: recipe.tags,
+    }
+})
+}
+
 fetchData().then(data => {
```
Comment (grimeyg) [inline src/scripts.js]: Clean function call and .thens

File: src/scripts.js (hunk 2 above)
Comment (grimeyg) [reply src/scripts.js]: We can do that along with making the filter options hidden as well. Once filter options are hidden there can be 4 cards in each row

**Pull request merged**